### PR TITLE
chore(docs): gatsby image - code block langue, fix snippets, fences a…

### PR DESCRIPTION
**Pull request by muescha**
## Description

changes:
- code block language to `jsx`
- fix snippets (close `` tags, add import for `Img`
- code fences around constants and names

## Related Issues

- #27945 `docs(gatsby-plugin-image): Update readme`

### CODE REVIEW

File: packages/gatsby-plugin-image/README.md [hunk 1]
```diff
@@ -316,31 +328,31 @@ The optional helper function `getImage` takes a file node and returns `file?.chi
...
 - **formats**: an array of file formats to generate. The default is `[AUTO, WEBP]`, which means it will generate images in the same format as the source image, as well as in the next-generation [WebP](https://developers.google.com/speed/webp) format. We strongly recommend you do not change this option, as doing so will affect performance scores.
 - **quality**: The default quality. This is overridden by any format-specific options
-- **blurredOptions**: Options for the low-resolution placeholder image. Set placeholder to "BLURRED" to use this
+- **blurredOptions**: Options for the low-resolution placeholder image. Set `placeholder` to `BLURRED` to use this
   - width
   - toFormat
-- **tracedSVGOptions**: Options for traced placeholder SVGs. You also should set placeholder to "SVG".
+- **tracedSVGOptions**: Options for traced placeholder SVGs. You also should set `placeholder` to `SVG`.
```
Comment (muescha) [inline packages/gatsby-plugin-image/README.md:355]: there is no `SVG` 
only `TRACED_SVG`

should this changed here or `SVG` added to `placeholder`?

**Comment by ascorbic** [comment]
Hi @muescha ,
Thanks for your help. I'll accept this, but bear in mind that we're rewriting all of the docs for these, so I wouldn't put more time into them right now.

File: packages/gatsby-plugin-image/README.md (hunk 1 above)
Comment (ascorbic) [reply packages/gatsby-plugin-image/README.md:356]:
```suggestion
- **tracedSVGOptions**: Options for traced placeholder SVGs. You also should set `placeholder` to `TRACED_SVG`.
```

File: packages/gatsby-plugin-image/README.md [hunk 2]
```diff
@@ -71,55 +71,67 @@ Because the images still need to be resized during build, the props must be able
 
 This does not work:
 
-```js
+```jsx
 // ⚠️ Doesn't work
 
-({ logo }) => <Img src={logo}>
+import React from "react"
+import Img from "gatsby-image"
```
Comment (ascorbic) [inline packages/gatsby-plugin-image/README.md:78]: This is not correct. The component is `StaticImage` from `"gatsby-plugin-image"`

**Comment by ascorbic** [comment]
OK, after reviewing I'm afraid I'm not going to merge it. There are a number of mistakes in the changes, as well as other outdated information in the old docs. I am going to update this with the new, correct docs soon. Thanks again for your help, but I'd hold off on working on this plugin until it's stable

**Comment by muescha** [comment]
ok - i wait until stable

**Pull request closed without merging**